### PR TITLE
CORE-6297 added doc for vnode commands

### PR DESCRIPTION
**Pull request by pmck91**

### CODE REVIEW

File: tools/plugins/virtual-node/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Virtual Node Cli Plugin
+
+Upload and overwrite earlier stored CPI record.
```
Comment (nadinequinr3) [inline tools/plugins/virtual-node/README.md]:
```suggestion
Uploads and replaces a previously stored CPI record.
```

File: tools/plugins/virtual-node/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Virtual Node Cli Plugin
```
Comment (nadinequinr3) [inline tools/plugins/virtual-node/README.md]:
```suggestion
# Virtual Node CLI Plugin
```

File: tools/plugins/virtual-node/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Upload and overwrite earlier stored CPI record.
+Any sandboxes running an overwritten version of CPI will be purged and optionally
```
Comment (nadinequinr3) [inline tools/plugins/virtual-node/README.md]:
```suggestion
The plugin purges any sandboxes running an overwritten version of a CPI and optionally
```

File: tools/plugins/virtual-node/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+vault data for the affected Virtual Nodes wiped out.
```
Comment (nadinequinr3) [inline tools/plugins/virtual-node/README.md]: Can we use something other than "wiped out" ? Deleted?
How is it optional? Is this a subcommand or configured somewhere?

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: tools/plugins/virtual-node/README.md (hunk 4 above)
Comment (vkolomeyko) [reply tools/plugins/virtual-node/README.md]: Yes, "deleted" would be fine.

There will be an additional sub-command which will allow choosing whether to delete the DB or not once @benmcmahon100 changes are merged.

**Review by vkolomeyko — DISMISSED** [review]
LGTM

File: tools/plugins/virtual-node/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Virtual Node CLI Plugin
+
+Uploads and replaces a previously stored CPI record.
+The plugin purges any sandboxes running an overwritten version of a CPI and optionally
+vault data for the affected Virtual Nodes deleted.
```
Comment (nadinequinr3) [inline tools/plugins/virtual-node/README.md]:
```suggestion
deletes vault data for the affected Virtual Nodes.
```

Comment (nadinequinr3) [reply tools/plugins/virtual-node/README.md]: We will need to add info about the sub-command

File: libs/virtual-node/virtual-node-endpoints-maintenance/src/main/kotlin/net/corda/libs/virtualnode/maintenance/endpoints/v1/VirtualNodeMaintenanceRPCOps.kt [hunk 6]
```diff
@@ -27,7 +27,7 @@ interface VirtualNodeMaintenanceRPCOps : RpcOps {
      * Even if CPI with the same metadata has already been previously uploaded, this endpoint will overwrite earlier
      * stored CPI record.
      * Furthermore, any sandboxes running an overwritten version of CPI will be purged and optionally vault data for
```
Comment (nadinequinr3) [inline libs/virtual-node/virtual-node-endpoints-maintenance/src/main/kotlin/net/corda/libs/virtualnode/maintenance/endpoints/v1/VirtualNodeMaintenanceRPCOps.kt]:
```suggestion
     * The plugin purges any sandboxes running an overwritten version of a CPI and optionally deletes vault data for the affected Virtual Nodes.
```

File: tools/plugins/virtual-node/src/main/kotlin/net/corda/cli/plugins/vnode/commands/ResetCommand.kt [hunk 7]
```diff
@@ -18,7 +18,7 @@ import kotlin.reflect.KClass
     name = "reset",
     description = ["Upload and overwrite earlier stored CPI record.",
         "Any sandboxes running an overwritten version of CPI will be purged and optionally",
```
Comment (nadinequinr3) [inline tools/plugins/virtual-node/src/main/kotlin/net/corda/cli/plugins/vnode/commands/ResetCommand.kt]:
```suggestion
        "The plugin purges any sandboxes running an overwritten version of a CPI and optionally ",
```"

File: tools/plugins/virtual-node/src/main/kotlin/net/corda/cli/plugins/vnode/commands/ResetCommand.kt [hunk 8]
```diff
@@ -18,7 +18,7 @@ import kotlin.reflect.KClass
...
-        "vault data for the affected Virtual Nodes wiped out."]
+        "vault data for the affected Virtual Nodes deleted."]
```
Comment (nadinequinr3) [inline tools/plugins/virtual-node/src/main/kotlin/net/corda/cli/plugins/vnode/commands/ResetCommand.kt]:
```suggestion
        "deletes vault data for the affected Virtual Nodes."]
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

**Review by nadinequinr3 — DISMISSED** [review]

**Review by nadinequinr3 — DISMISSED** [review]

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Pull request merged**